Mark fields as readonly in docs

jbolinger opened the pull request:
Marks the attribute as readonly in the generated docs if the `OUTPUT_ONLY` flag is set on the field.

I also added a field presenter test. Happy to add more tests as part of this PR, but I'm looking for advice on the structure. Should we have a folder under `test/gapic/presenter/field`, keep it with the method-level tests as I've done here since those presenters are fairly related, etc.? Alternatively, I can defer to #173 if you'd prefer to wait on adding these.

[resolves #103]

quartzmo commented:
> Should we have a folder under `test/gapic/presenter/field`, keep it with the method-level tests as I've done here since those presenters are fairly related, etc.?

I see that we have `/Users/quartzmo/code/google/codez/gapic-generator-ruby/gapic-generator/test/gapic/presenters/method_test.rb` which contains `class MethodPresenterTest`. Personally I think that could be renamed to `presenters/method_presenter_test.rb`. Your test could be added in the same directory as `presenters/field_presenter_test.rb`.

One note, however, is that I might have expected the presenters path to be preserved:

`templates/default/helpers/presenters/method_presenter.rb`

and

`test/templates/default/helpers/presenters/method_presenter_test.rb`

But we've done a lot of moving things around already so don't take that as a very serious recommendation, just a nit.

jbolinger commented:
Thanks @quartzmo - I moved the test alongside the method presenter test for consistency for now. We can always move it again later. 

I also added a garbage test, but it doesn't cover all the bases yet. In particular, I couldn't figure out how to get the nested protos to pass but I think that can be addressed later.

quartzmo commented:
Is anything else needed for this PR? I approved yesterday.

@blowmage Want to take a look?
